### ISSUE TITLE
Deploying nodejs-ex into OpenShift 4.4 does not work

### ISSUE THREAD

**Issue by phracek** [sclorg/nodejs-ex#236]
Commands used for deploying image:
```bash
$ oc process -f https://raw.githubusercontent.com/sclorg/nodejs-ex/master/openshift/templates/nodejs.json | oc apply -f -

service "nodejs-example" created
route.route.openshift.io "nodejs-example" created
imagestream.image.openshift.io "nodejs-example" created
buildconfig.build.openshift.io "nodejs-example" created
deploymentconfig.apps.openshift.io "nodejs-example" created
```

OCP status:
```bash
$ oc status
In project default on server https://api.ocp4.ocp.local:6443

svc/kubernetes - 172.30.0.1:443 -> 6443

http://nodejs-example-default.apps.ocp4.ocp.local (svc/nodejs-example)
  dc/nodejs-example deploys istag/nodejs-example:latest <-
    bc/nodejs-example source builds https://github.com/sclorg/nodejs-ex.git on openshift/nodejs:8
      not built yet
    deployment #1 waiting on image or update

2 warnings identified, use 'oc status -v' to see details.
```

starting build:
```bash
$ oc start-build nodejs-example --follow
The ImageStreamTag "nodejs:8" is invalid: from: Error resolving ImageStreamTag nodejs:8 in namespace openshift: unable to find latest tagged image
```

@hhorak @pkubatrh CC

**Comment by bparees** [comment]
The template should be updated to use nodejs:12
https://github.com/sclorg/nodejs-ex/blob/master/openshift/templates/nodejs.json#L238

looks like this was already done for the nodejs-mongo template, but the nodejs template was overlooked:
https://github.com/sclorg/nodejs-ex/blob/master/openshift/templates/nodejs-mongodb.json#L439

**Comment by bparees** [comment]
https://github.com/sclorg/nodejs-ex/pull/237
